subsys/bluetooth/mesh/shell/shell.c

```diff
@@ -877,16 +877,16 @@ static int cmd_auth_method_set_output(const struct shell *sh, size_t argc, char
 static int cmd_auth_method_set_static(const struct shell *sh, size_t argc, char *argv[])
 {
 	size_t len;
-	uint8_t static_val[16];
+	uint8_t static_values[16];
```

Comment:
This is a single static OOB value. I'd recommend then `static_oob` or `static_oob_auth` instead.

Reply:
Renamed.